modules/light-clients/07-tendermint/client_state.go

```diff
@@ -218,6 +218,15 @@ func (cs ClientState) VerifyMembership(
 	path exported.Path,
 	value []byte,
 ) error {
+	// check if there is a proof passed in
+	if height.IsZero() {
+		return sdkerrors.Wrap(sdkerrors.ErrInvalidHeight, "proof height must be non-zero")
+	}
+
+	if len(proof) == 0 {
+		return sdkerrors.Wrap(commitmenttypes.ErrInvalidProof, "cannot submit an empty proof")
+	}
```

Comment:
it looks like we removed test cases which checked for these in ValidateBasic, we should probably add some test cases that cause these errors in the `Verify(Non)MemberShip` functions. WDYT?

Reply:
Agreed